civis/tests/test_io.py

```diff
@@ -419,3 +433,24 @@ def test_split_schema_tablename_raises():
     s = "table.with.too.many.periods"
     with pytest.raises(ValueError):
         civis.io._tables.split_schema_tablename(s)
+
+
+@mock.patch.object(civis.io._tables, '_sql_script', autospec=True,
+                   return_value=[-1000, 1000])
+@mock.patch.object(civis.io._tables, 'APIClient', autospec=True)
```

Comment:
Rather than patching the `APIClient` constructor, you could pass the mock client as an input parameter to `export_to_civis_file`.

Reply:
Cool, that probably is easier to understand.